Formula/p/php-intl.rb

```diff
@@ -0,0 +1,47 @@
+class PhpIntl < Formula
+  desc "PHP internationalization extension"
+  homepage "https://www.php.net/manual/en/book.intl.php"
+  url "https://www.php.net/distributions/php-8.4.12.tar.xz"
+  mirror "https://fossies.org/linux/www/php-8.4.12.tar.xz"
+  sha256 "c1b7978cbb5054eed6c749bde4444afc16a3f2268101fb70a7d5d9b1083b12ad"
+  license "PHP-3.01"
+  head "https://github.com/php/php-src.git", branch: "master"
+
+  livecheck do
+    formula "php"
+  end
+
+  depends_on "pkgconf" => :build
+  depends_on "icu4c@77"
+  depends_on "php"
+
+  def install
+    extension_dir = Utils.safe_popen_read("php-config", "--extension-dir").chomp
+    extension_dir = lib/"php/pecl/#{File.basename(extension_dir)}"
```

Comment:
Still need to figure this out. Not sure how brew PHP is currently set up for extensions.

For now, I added the ext-intl.ini that automatically loads extension on install (and unloads it on `brew unlink php-intl` or uninstall).

Reply:
Directory should be correct, i.e. we modify php.ini config so `dl()` loads from:
```
/opt/homebrew/lib/php/pecl/20240924/
```

Only thing is to make sure this exists earlier on. Perhaps mkdir during php postinstall. Trying to add file in php bottle will just result in another symlink.

Other part is figuring out best way to handle loading, i.e. add config to always load or not. Given our previous approach was to statically link into PHP binary, it may be better to always load.

Reply:
Will update ini file to just `extension=intl`. Fedora's `php-intl` package has same (with comment):
```console
$ cat etc/php.d/20-intl.ini
; Enable intl extension module
extension=intl
```